source/optional_features/use_your_own_error_pages/index.html.md.erb

```diff
@@ -0,0 +1,50 @@
+---
+title: Use your own payment failure pages 
+weight: 135
+---
+
+# Use your own payment failure pages
+
+You can [contact us](/support_contact_and_more_information/#contact-us) to set
+this feature up for your service. 
+
+Terminal, unsuccessful payment states are:
+
+* payment declined
+* payment cancelled
+* payment expired
+* error (eg technical error)
+
+If you use this feature, when a payment reaches one of these states, GOV.UK
```

Comment:
in some categories of error, we won't be able to redirect. however this is extremely rare (most likely is NAXSI errors). in routine errors and payment failures we'll be able to redirect. so it's probably fine to leave it like this!

Reply:
There's probably no real problem adding that in rare cases we won't be able to redirect, as long as you're OK adding a note to "Contact us" if that happens?

Reply:
sure